engine/runtime-compiler/src/test/java/org/enso/compiler/test/mock/VirtualFileSystemTest.java

```diff
@@ -0,0 +1,162 @@
+package org.enso.compiler.test.mock;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.Assert.fail;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.nio.charset.Charset;
+import org.apache.commons.vfs2.FileObject;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class VirtualFileSystemTest {
+  @ClassRule public static final TemporaryFolder TMP_DIR = new TemporaryFolder();
```

Comment:
Why do we need temporary folder? Is the variable even used?

Reply:
It was a leftover. Removed in 116cbcc6994ae5260a1e91d174fc50283bc81227